libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -59,4 +46,4 @@ export class BookSearchComponent implements OnInit {
       this.store.dispatch(clearSearch());
```

Comment:
At line no 44, instead of using `this.searchTerm`, you can directly use `this.searchForm.value.term`.
If you do so, you can remove below code from line no 29.

```
get searchTerm(): string {
    return this.searchForm.value.term;
 }
```